packages/coil-extension/src/background/services/AuthService.ts

```diff
@@ -182,16 +183,24 @@ export class AuthService extends EventEmitter {
 
     this.log('updateWhoAmi resp', resp.data)
     if (resp.data?.whoami) {
-      this.store.user = {
-        ...resp.data.whoami
-      }
+      this.store.user = resp.data.whoami
 
       // Data needed for tipping
       // tipping-beta: featureEnabled: boolean
       // minimum tip limit: minTipLimit > minTipLimit
       // remaining daily amount: whoami > tipping > limitRemaining
 
-      this.formatTipSettings(token)
+      if (this.store.user) {
+        this.store.user = {
+          ...this.store.user,
+          tipSettings: await formatTipSettings(
```

Comment:
I'm wondering if instead of having the `formatTipSettings` here can we make the call to the tipping service for `updateTipSettings`?

Reply:
@mwill8886 whats the pro vs con of doing that? Is it just one more api call but the data will be cleaner? Do you need a backend update for this?